assets/javascripts/needleeditor.js

```diff
@@ -504,7 +535,19 @@ function setup_needle_editor(imageurl, default_needle)
     return true;
   });
 
-  $('#property_workaround').click(function() { nEditor.changeProperty(this.name, this.checked); });
+  $('#property_workaround').click(function() {
+    nEditor.changeProperty(this.name, this.checked);
+
+    if (this.checked) {
+      $("#workaround_reason_div").show();
```

Comment:
You can simply [.toggle()](https://api.jquery.com/toggle/) here

Reply:
And hope it never gets out-of-sync with the checkbox state :-)

Reply:
What's the alternative? A click that doesn't do anything? If a click won't effectively toggle there's a bug anyway.